pkg/executor/server.go

```diff
@@ -91,7 +92,8 @@ func NewServer(host HostInterface, enableDebuggingHandlers bool, ns string) Serv
 func (s *Server) InstallDefaultHandlers() {
 	healthz.InstallHandler(s.mux)
 	profile.InstallHandler(s.mux)
-	s.mux.HandleFunc("/podInfo", s.handlePodInfo)
+	s.mux.HandleFunc("/podInfo", s.handlePodInfoOld)
+	s.mux.HandleFunc("/api/v1beta1/podInfo", s.handlePodInfoVersioned)
```

Comment:
k8s code: I don't know why they use v1beta1 instead of v1beta2, but last I checked those are the only two "publishable" types that clients can use for interacting with the API.